### PR TITLE
Add additional metadata

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary
Screenshot from 2020-03-31 19-10-06

#### Ticket Link
Fixes https://github.com/mattermost/mattermost-plugin-skype4business/issues/44

### CODE REVIEW

File: plugin.json [hunk 1]
```diff
@@ -2,7 +2,11 @@
   "id": "skype4business",
   "name": "Skype for Business",
   "description": "Skype for Business plugin for Mattermost 5.2+.",
+  "homepage_url": "https://github.com/mattermost/mattermost-plugin-skype4business",
+  "support_url": "https://github.com/mattermost/mattermost-plugin-skype4business/issues",
+  "icon_path": "assets/profile.svg",
   "version": "0.1.2",
+  "min_server_version": "5.10.0",
```
Comment (jfrerich) [inline plugin.json]: What is the process for choosing the `min_server_version`?

Comment (hanzei) [reply plugin.json]: Good question. The "latest" API method I could found was `GetBundlePath`. @kosgrz Could you double check that I didn't missed anything?

Comment (kosgrz) [reply plugin.json:9]: @hanzei I've checked all APIs methods and `GetBundlePath` is the newest one :+1:

**Comment by hanzei** [comment]
@DHaussermann If you deploy this branch to your machine and open the Marketplace, you should see the changes. Let me know if you need help

**Comment by hanzei** [comment]
Skipping QA review per https://community.mattermost.com/core/pl/k1js57wa8tnr5qtrzturas7txh

**Pull request merged**